apps/auth/src/pages/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { signIn, signOut, useSession } from 'next-auth/react'
+
+import { Button, H1 } from '@uzh-bf/design-system'
+import Image from 'next/image'
+import { useRouter } from 'next/router'
+import Footer from 'shared-components/src/Footer'
+
+function SignInOutButton() {
+  const router = useRouter()
+
+  const { data: session } = useSession()
+
+  if (session) {
+    return (
+      <>
```

Comment:
this is not really necessary, as we will handle logout from the other apps, but does not hurt either